softvis3d.properties

```diff
@@ -9,8 +9,9 @@ defaults.mavenGroupId=de.rinderle.softvis3d
 defaults.mavenArtifactId=softvis3d
 
 1.3.0.description=Compatibility with SQ 10.X.
-1.3.0.sqVersions=[10.0,LATEST]
-1.3.0.date=2023-09-18
+1.3.0.sqs=[10.8,LATEST]
```

Comment:
date is missing here

Reply:
Actually, it's pushed onto the next line. But thanks for the review